src/sshd/install.sh

```diff
@@ -93,35 +93,6 @@ sed -i -E "s/#*\s*Port\s+.+/Port ${SSHD_PORT}/g" /etc/ssh/sshd_config
 # Need to UsePAM so /etc/environment is processed
 sed -i -E "s/#?\s*UsePAM\s+.+/UsePAM yes/g" /etc/ssh/sshd_config
 
-# Script to store variables that exist at the time the ENTRYPOINT is fired
```

Comment:
Can you bump the patch version here? Thanks!  
https://github.com/devcontainers/features/blob/c9f622064b3c009b4101a7e9803148ff25d66ed3/src/sshd/devcontainer-feature.json#L3

Reply:
done